core/Makefile

```diff
@@ -2,7 +2,7 @@
 .SUFFIXES:
 #---------------------------------------------------------------------------------
 ifeq ($(SDKDIR),)
-SDKDIR = ../sdk
+SDKDIR = $(CURDIR)/../sdk
```

Comment:
does `$(CURDIR)` change when it goes over the makefile a second time to do the actual build?

Reply:
It seems to pick up `SDKDIR` from the first `make` actually
Adding `@echo $(origin SDKDIR) $(SDKDIR) $(CURDIR)` to two rules gives this:
```
file /home/thepikachugamer/Desktop/WiiHB/StarStruck/core/../sdk /home/thepikachugamer/Desktop/WiiHB/StarStruck/core
environment /home/thepikachugamer/Desktop/WiiHB/StarStruck/core/../sdk /home/thepikachugamer/Desktop/WiiHB/StarStruck/core/build
```

Reply:
...okay, the CI seems to have a whole different story